Update index.mdx (#199)
hashicorp/terraform-docs-common: website/docs/plugin/index.mdx

- Learn more about [how Terraform Core interacts with plugins](/plugin/how-terraform-works).
- Learn the [design principles](/plugin/hashicorp-provider-design-principles) HashiCorp developers follow when creating providers.
- Decide [which SDK](/plugin/which-sdk) is right for your provider.

> - Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](/terraform/tutorials/providers) and [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework)

Author: ```suggestion
- Try these hands-on tutorials: [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework) and [Call APIs with Terraform Providers (SDKv2)](/terraform/tutorials/providers) 
```

> - Try these hands-on tutorials: [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework) and [Call APIs with Terraform Providers (SDKv2)](/terraform/tutorials/providers) 

Author: ```suggestion
- Try these hands-on tutorials: [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework) and [Call APIs with Custom SDK Providers](/terraform/tutorials/providers) 
```

> - Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) and [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-resource-create)

Reviewer: Two questions:

- Should we adjust the other Learn link to be relative?
- Should we point to the framework collection rather than a specific tutorial?

Maybe something like:

```suggestion
- Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](/terraform/tutorials/providers) and [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework)
```

Reviewer: We could (or maybe should?) flip the order of these links as well to potentially encourage framework slightly more over sdk/v2.

Author: oooooo i like that! let me do it real quick
